src/modules/navigator/rtl.cpp

```diff
@@ -80,11 +80,23 @@ void RTL::on_inactive()
 	if ((hrt_absolute_time() - _destination_check_time) > 1_s) {
 		_destination_check_time = hrt_absolute_time();
 
-		if (_navigator->home_global_position_valid()) {
+		const vehicle_global_position_s &global_position = *_navigator->get_global_position();
+
+		const bool global_position_recently_updated = global_position.timestamp > 0
+				&& hrt_elapsed_time(&global_position.timestamp) < 10_s;
+
+		rtl_time_estimate_s rtl_time_estimate{};
+		rtl_time_estimate.valid = false;
```

Comment:
The `{}` already initializes everything to 0/false but you can leave it like that to be more explicit

Reply:
Yes that was my thinking as well